### ISSUE TITLE
Broken link to "OWASP_Speakers_Project"

### ISSUE THREAD

**Issue by hotpheex** [OWASP/www-policy#11]
The link to `https://www.owasp.org/index.php/Category:OWASP_Speakers_Project` in `guidebook/chapter-leader.md` throws a 404.

**Comment by kingthorin** [comment]
@knoblochmartin is https://wiki.owasp.org/index.php/Category:OWASP_Speakers_Project still a thing? I was unable to find any up-to-date content.

**Comment by knoblochmartin** [comment]
Short answer, "no, I don't think so"
Long answer:  Never really picked up and died silently.
When I joined the BoD, I did transfer all my project to new leaders. Not all volunteers who agreed to be the new project leaders actually picked up the activities. Not sure who volunteered to take over this project.

Still think, definitely with the current virtual meetings, this would be a valuable resource for chapter to find speakers!

Cheers,
-martin
